code/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -613,30 +610,30 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "breathing-float",
+   "id": "framed-string",
    "metadata": {
     "kernel": "SoS"
    },
    "outputs": [],
    "source": [
     "[qc_3 (export to PLINK)]\n",
     "parameter: walltime = '24h'\n",
+    "parameter: remove_duplicates = False\n",
     "output: f'{_input:nn}.bed'\n",
     "task: trunk_workers = 1, trunk_size = job_size, walltime = walltime, mem = mem, cores = numThreads, tags = f'{step_name}_{_output:bn}'\n",
     "bash: container = container, expand= \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout'\n",
-    "    plink --vcf ${_input} \\\n",
+    "    plink2 --vcf ${_input} \\\n",
```

Comment:
@hsun3163 if you review other changes in VCF_QC.ipynb you'll see that you are overriding some of the changes I made. Please download the current `master` of VCF_QC.ipynb and only make changes at this particular step.